opt_PXL.ino

```diff
@@ -22,7 +23,7 @@ byte r = 0;                                // Variable for the Red Value
 #define STRIP_LENGTH 144                  // Set the number of LEDs the LED Strip
 byte frameDelay = 15;                      // default for the frame delay 
 int frameBlankDelay = 0;                  // Default Frame blank delay of 0
-byte menuItem = 1;                         // Variable for current main menu selection
+char menuItem = 1;                         // Variable for current main menu selection

```

Comment:
Intended change?

Reply:
It was an intended change in order to save some memory space but can probably be put back as it

Reply:
I'd put it back, just to be sure.